test.js

```diff
@@ -221,3 +221,15 @@ test('properly rounds milliseconds with secondsDecimalDigits', t => {
 	t.is(fn((3600 * 1e3) - 1), '1 hour');
 	t.is(fn((2 * 3600 * 1e3) - 1), '2 hours');
 });
+
+test('work with colon output option', t => {
+	t.is(prettyMilliseconds(1000, {colonNotation: true}), '0:01');
```

Comment:
Why is minute zero prefixed, but hour is not?

Reply:
I tried googling for an official standard outlining how this format should behave, but was unfortunately unable to find one. However, the attached screenshot from the Kona Ironman World Championships this year shows what the commonly accepted convention is:
![image](https://user-images.githubusercontent.com/33317356/68549745-51fae980-0404-11ea-8073-95f7adbd96ab.png)
Here we can see, that Jan Frodeno's total time `2h 24m 45s` was `1s` behind whoever was leading at that time, which in this notation would be expressed as `2:24:45` and `0:01` respectively. This example doesn't show fractional seconds, but for seconds regular decimal formatting applies.

Reply:
I agree with your assessment.